### PR TITLE
Create the ruby-sys crate

### PR DESCRIPTION
**Pull request by steveklabnik**
This is a big PR, but it's not _that_ big.

Cargo has a convention of [`-sys` packages](http://doc.crates.io/build-script.html#-sys-packages). The idea is that the low-level stuff should be a package, and then higher-level bindings should be built on top of it.

ruru _is_ that kind of package, and `src/unsafe_bindings` is really a `-sys` crate. So, I made one. :smile: 

I went ahead and pushed it up, so that nobody else could squat it while we discuss this PR: https://crates.io/crates/ruby-sys If this PR is accepted, I'll make you an owner of the package as well. If you don't want this PR, well, `ruby-sys` should still exist, imho :smile:

### CODE REVIEW

**Comment by steveklabnik** [comment]
... well apparently this fails the tests. I was just building my own project, I forgot `cargo test` :sweat_smile: 

So, I will fix it tonight or tomorrow, but still worth discussing in the meantime :)

**Comment by steveklabnik** [comment]
(for an example of a crate structured like this, https://github.com/sfackler/rust-openssl )

File: ruby-sys/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ruby-sys"
+version = "0.1.1"
+authors = ["Steve Klabnik <steve@steveklabnik.com>",
+           "Dmitry Gritsay <unseductable@gmail.com>"]
+description = "Low level bindings to MIR"
```
Comment (malept) [inline ruby-sys/Cargo.toml]: That's supposed to be "Low level bindings to MRI", right?

**Comment by d-unseductable** [comment]
Wow! Awesome work, @steveklabnik! Do you think we can merge it?

File: ruby-sys/Cargo.toml (hunk 1 above)
Comment (steveklabnik) [inline ruby-sys/Cargo.toml]: ah thanks! Too much Rust, hehe

**Comment by steveklabnik** [comment]
@d-unseductable awesome :smile:  Now that I have fixed the bug @malept has found out, feel free to merge once Travis lands. After you do so, I will add you to the crate on crates.io.

**Comment by d-unseductable** [comment]
Thank you very much!

**Comment by steveklabnik** [comment]
@d-unseductable woo! It appears that `unseductable@gmail.com` isn't your login email on crates.io though, maybe DM me the one you use?

**Comment by d-unseductable** [comment]
@steveklabnik, just double checked my crates.io profile! `unseductable@gmail.com` is the email used for my account. Do you get any errors regarding the email address?

**Comment by steveklabnik** [comment]
I mis-read the erorr message, you need the login name, not email. You should have just been added.

**Comment by d-unseductable** [comment]
Yes it works, thanks! Also if you want I can add you to this repo's collaborators, what do you think?

**Comment by steveklabnik** [comment]
Sure, if you'd like :) I'd like to do some more work on this, but I also am interested in how you feel about things. I will probably open some issues for discussions later in the week.

**Pull request merged**